coreneuron/network/partrans.cpp

```diff
@@ -133,12 +133,9 @@ void nrnthread_v_transfer(NrnThread* _nt) {
 void nrn_partrans::copy_gap_indices_to_device() {
     // Ensure index vectors, src_gather, and insrc_buf_ are on the gpu.
     if (insrcdspl_) {
-        int n_insrc_buf = insrcdspl_[nrnmpi_numprocs];
-        static_cast<void>(n_insrc_buf);
-        nrn_pragma_acc(enter data create(insrc_buf_[:n_insrc_buf]))
-        // clang-format off
-        nrn_pragma_omp(target enter data map(alloc: insrc_buf_[:n_insrc_buf]))
-        // clang-format off
+        // TODO: we don't actually need to copy here, just allocate + associate
+        // storage on the device
+        cnrn_target_copyin(insrc_buf_, insrcdspl_[nrnmpi_numprocs]);
```

Comment:
change needed to keep our private present table up to date.
could consider adding `cnrn_target_allocate` or similar.